backend-go/internal/infra/storage/client.go

```
@@ -0,0 +1,15 @@
+package storage
+
+import (
+	"context"
+	"time"
+)
+
+const (
+	UrlExpirationTime = 15 * time.Minute
```

golangci-lint (1.64.8)

[warning] var-naming: const UrlExpirationTime should be URLExpirationTime

(revive)